src/features/sub-tasks-progress/BoardSettings/ColorSchemeSettings/StatusCategorySettings.tsx

```
@@ -0,0 +1,77 @@
+          <>
+            {status.name}
+            <span
+              style={{
+                display: 'inline-block',
+                width: '1em',
+                height: '1em',
+                backgroundColor: mapStatusCategeoryToColor(status.statusCategory),
+                marginRight: '0.5em',
+              }}
+            />
+            <Checkbox
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)